bowlingScore.js

```diff
@@ -0,0 +1,54 @@
+class BowlingScore {
+  constructor() {
+    this.totalScore = 0;
+    this.scorecard = [];
```

Comment:
Just as an addition to this, it seems like the main use of the `.totalscore` variable is in calculating the score, and it's set to zero before calculating it. In this case, it's better to use arguments and return values to move data between functions, instead of instance variables.